### PR TITLE
Love-1260: polish parameter names

### PR DESCRIPTION
**Pull request by eltonkevani**
## Definition of Done

**General**
 - [ ] Branch is up-to-date with beta
 - [ ] Blueprint can be tested by Jenkins
 - [ ] Code is reviewed and tested by someone else in the team

**Testing**
- [ ] Works on Linux, Windows and Mac
- [ ] Works on all supported K8s flavors (Docker Desktop for mac/Windows, AWS EKS, Plain multinode K8s)
- [ ] Functionality is manually tested where not possible to automate
- [ ] Automated unit tests added and are green
- [ ] Automated integration tests added where needed and are green

### CODE REVIEW

**Comment by ilx** [comment]
Why Polish? Use Hungarian parameter names... ;)

**Comment by eltonkevani** [comment]
> Why Polish? Use Hungarian parameter names... ;)

true :-D

File: xl-infra/blueprint.yaml [hunk 1]
```diff
@@ -114,15 +115,15 @@ spec:
     - name: K8sClientCert
       type: SecretEditor
       description: "Using the Kubernetes client certificate from your context"
-      value: !expr "UseKubeconfig && K8sSetup != 'AwsEKS' && K8sAuthentication == 'UserInput' ? k8sConfig('UserClientCertificateData') : ''"
+      value: !expr "UseKubeconfig && K8sSetup != 'AwsEKS' && K8sAuthentication != 'FilePath' && K8sAuthentication != 'Token'? k8sConfig('UserClientCertificateData') : ''"
```
Comment (deepu105) [inline xl-infra/blueprint.yaml]: @eltonkevani I dont get this, why did you change this `K8sAuthentication != 'FilePath' && K8sAuthentication != 'Token'` is same as `K8sAuthentication == 'UserInput'`

Comment (eltonkevani) [reply xl-infra/blueprint.yaml]: i though the same but is not, it has to do with how default works in select params, default is empty
